octobot_trading/exchanges/config/ccxt_exchange_ui_settings.py

```diff
@@ -4,105 +4,114 @@
-        )
-        settings.GET_MY_RECENT_TRADES_METHODS = exchange.UI.user_input(
-            "get_my_recent_trades_methods",
-            common_enums.UserInputTypes.MULTIPLE_OPTIONS,
-            def_val=settings.GET_MY_RECENT_TRADES_METHODS,
-            options=settings.ALL_GET_MY_RECENT_TRADES_METHODS,
-            registered_inputs=inputs,
-            title="Get my recent trades methods: Each method will be tried until OctoBot find trades",
-            parent_input_name=experimental_settings_name,
-        )
-        settings.CANDLE_LOADING_LIMIT = exchange.UI.user_input(
-            "candle_loading_limit",
-            common_enums.UserInputTypes.INT,
-            def_val=settings.CANDLE_LOADING_LIMIT,
-            registered_inputs=inputs,
-            title="Candle pagination limit (0 is disabled): Many exchanges have a limit of how many candles we can fetch at once. "
-            "(Hint: OctoBot will still be able to download candle history and automatically split it up into multiple requests)",
-            parent_input_name=experimental_settings_name,
-        )
-        settings.MAX_RECENT_TRADES_PAGINATION_LIMIT = exchange.UI.user_input(
-            "max_recent_trades_pagination_limit",
-            common_enums.UserInputTypes.INT,
-            def_val=settings.MAX_RECENT_TRADES_PAGINATION_LIMIT,
-            registered_inputs=inputs,
-            title="Recent trades pagination limit (0 is disabled): Many exchanges have a limit of how many trades we can fetch at once. "
-            "(Hint: OctoBot will still be able to download trades and automatically split it up into multiple requests)",
-            parent_input_name=experimental_settings_name,
-        )
-        settings.MAX_ORDER_PAGINATION_LIMIT = exchange.UI.user_input(
-            "max_order_pagination_limit",
-            common_enums.UserInputTypes.INT,
-            def_val=settings.MAX_ORDER_PAGINATION_LIMIT,
-            registered_inputs=inputs,
-            title="Orders pagination limit (0 is disabled): Many exchanges have a limit of how many orders we can fetch at once. "
-            "(Hint: OctoBot will still be able to download orders and automatically split it up into multiple requests)",
-            parent_input_name=experimental_settings_name,
-        )
-        settings.MARKET_STATUS_FIX_PRECISION = exchange.UI.user_input(
-            "market_status_fix_precision",
-            common_enums.UserInputTypes.BOOLEAN,
-            def_val=settings.MARKET_STATUS_FIX_PRECISION,
-            registered_inputs=inputs,
-            title="Fix market status precision: ",  # todo
-            parent_input_name=experimental_settings_name,
-        )
-        settings.MARKET_STATUS_REMOVE_INVALID_PRICE_LIMITS = exchange.UI.user_input(
-            "MARKET_STATUS_REMOVE_INVALID_PRICE_LIMITS",
+    settings: ccxt_exchange_settings.CCXTExchangeConfig = exchange.CONNECTOR_SETTINGS
+    if not settings.is_fully_tested_and_supported():
```

Comment:
In my opinion it should be checked before calling `initialize_experimental_exchange_settings`

Reply:
i moved to the exchange code